lib/dogma/util/script_strings.ex

```diff
@@ -7,98 +7,106 @@ defmodule Dogma.Util.ScriptStrings do
   for Elixir code and reporting errors when there should be none.
   """
 
+  @sigil_delimiters [{"|", "|"}, {~S("), ~S(")}, {"'", "'"}, {"(", ")"},
+                      {"[", "]"}, {"{", "}"}, {"<", ">"}]
+  @all_string_sigils @sigil_delimiters
+                      |> Enum.flat_map(fn({b, e}) ->
+                          [{"~s#{b}", e}, {"~S#{b}", e}]
```

Comment:
The way I do it in my project is that I have two modules `Code.Strings` and `Code.Sigils` which handle these things independently. This is easy to comprehend and test and not much of on overhead, since you only seem to use the functionality in a single place.

Anyhow. This PR stands as it is, since it provides what it says in the title and improves the functionality of a given module. I would rather not change course mid-PR. :+1: